automation/BrowserManager.py

```diff
@@ -529,39 +532,49 @@ def BrowserManager(
                 )
                 status_queue.put("OK")
             except WebDriverException:
+                status_obj = Status()
                 # We handle WebDriverExceptions separately here because they
                 # are quite common, and we often still have a handle to the
                 # browser, allowing us to run the SHUTDOWN command.
-                tb = traceback.format_exception(*sys.exc_info())
-                if "about:neterror" in tb[-1]:
-                    status_queue.put(("NETERROR", pickle.dumps(sys.exc_info())))
+                string_tb = traceback.format_exception(*sys.exc_info())
+                if "about:neterror" in string_tb[-1]:
+                    status_obj.set_name("NETERROR")
+                    status_queue.put(pickle.dumps(status_obj))
```

Comment:
Since the queue uses pickle internally, we can do 
```suggestion
                    status_queue.put(status_obj)
```